_includes/v22.2/metric-names-serverless.md

```diff
@@ -136,6 +136,13 @@ Name | Description
 `range.raftleadertransfers` | Number of raft leader transfers
 `range.removes` | Number of range removals
 `range.snapshots.generated` | Number of generated snapshots
+`range.snapshots.recv-in-progress` | Number of non-empty snapshots in progress on a receiver store
+`range.snapshots.recv-queue` | Number of queued non-empty snapshots on a receiver store
+`range.snapshots.recv-total-in-progress` | Number of empty and non-empty snapshots in progress on a receiver store
+`range.snapshots.send-in-progress` | Number of non-empty snapshots in progress on a sender store
+`range.snapshots.send-queue` | Number of queued non-empty snapshots on a sender store
+`range.snapshots.send-total-in-progress` | Number of empty and non-empty in-progress
+snapshots on a sender store
```

Comment:
Looks like an extra line break above this.

Reply:
I'm not sure what you mean. I don't see any extra breaks in the source (based on looking at the line numbers) or in https://deploy-preview-15608--cockroachdb-docs.netlify.app/docs/v22.2/ui-custom-chart-debug-page.html.

Reply:
I'm referring to these lines: 
<img width="566" alt="image" src="https://user-images.githubusercontent.com/8740013/201785059-0dfa2445-1511-4111-855e-764d6dc8f90e.png">